static/app/utils/replays/types.tsx

```diff
@@ -75,9 +75,7 @@ export function isBreadcrumbFrame(
   return Boolean(frame && 'category' in frame && frame.category !== 'issue');
 }
 
-export function isFeedbackFrame(
-  frame: ReplayFrame | undefined
-): frame is BreadcrumbFrame {
```

Comment:
We can be more specific than `frame is BreadcrumbFrame` to make the type narrowing string. Seems like this was a typo/copy-paste problem